helpers/useRedirectUnauthenticated.ts

```diff
@@ -0,0 +1,20 @@
+import { useRouter } from 'next/router'
+import { LOGIN_PATH } from '../constants'
+
+/**
+ * Redirects to login page if user is not authenticated.
+ * With next query param to redirect back to this page after login.
+ * @param {boolean} shouldRedirect Condition for redirection
+ */
+const useRedirectUnauthenticated = (shouldRedirect: boolean) => {
+  const router = useRouter()
```

Comment:
Only declare this variable if the variable `shouldRedirect` is true. Otherwise its pointless declaring it

Reply:
Makes sense.